backend/src/test/kotlin/org/loculus/backend/controller/submission/ReviseEndpointTest.kt

```diff
@@ -155,6 +155,31 @@ class ReviseEndpointTest(
             )
     }
 
+    @Test
```

Comment:
To test the chunk issue, we might want to add another test that sets chunk size to n=1 :)